kokoro/ndk-build/build.sh

```diff
@@ -30,9 +30,9 @@ unzip -q ninja-linux.zip
 export PATH="$PWD:$PATH"
 
 # Get Android NDK.
-wget -q https://dl.google.com/android/repository/android-ndk-r18b-linux-x86_64.zip
-unzip -q android-ndk-r18b-linux-x86_64.zip
-export ANDROID_NDK=$PWD/android-ndk-r18b
+wget -q https://dl.google.com/android/repository/android-ndk-r21d-linux-x86_64.zip
```

Comment:
NDK 21d is used by the CI flows in related projects; it's built into the docker image and will save this download.